src/docs/content/observability/observability-example/usecase.md

```diff
@@ -54,16 +54,20 @@ Send some more HTTP Status 406 requests (By placing orders above $105 as discuss
 
 ### Analyze the logs
 
-Navigate to ```Agent Configurations -> <your_agent_config> -> Explore log```
+Navigate to ```Logging -> Search``` and navigate through all the logs.
 
-![agent-logs](../../images/agent-logs.png)
+For this example, Switch to Advanced mode:
 
+Enter the following query
+```
+search "ocid1.compartment.oc1..xxxx/ocid1.loggroup.oc1.phx.xxxx/ocid1.log.oc1.phx.xxxx" |  logContent='*Payment declined*' and subject='/var/log/containers/*_mushop_*.log' 
+```
 You will see a failure in the logs as below:
 ```Payment declined: amount exceeds 105.00```
 
 ![json-log-details](../../images/json-log-details.png)
 
-Note: Notice the logContent.data which are nicely formatted based on our regex expression we provided during agent configuration (Advanced parser).
+Note: Notice the logContent.data which are nicely formatted based on our json parser we used during agent configuration.
```

Comment:
I would reword this to something like ... nicely formatted by the JSON parser used during the agent configuration